src/services/localSSHService.ts

```diff
@@ -60,6 +65,10 @@ export class LocalSSHService extends Disposable implements ILocalSSHService {
             this.isSupportLocalSSH = true;
         } catch (e) {
             this.logService.error(e, 'failed to copy local ssh client.js');
+            if (e.message) {
+                e.message = `Failed to copy local ssh client.js: ${e.message}`;
```

Comment:
should not we always set it to make easy to identify in error reporting?

Reply:
The error been caught will be something like: failed to copy-file / chmod / config ssh, to make it more clear, I add this prefix.